src/styles/themes/default.js

```diff
@@ -93,6 +93,7 @@ const oldTheme = {
     placeholderText: colors.gray3,
     heroCard: colors.blue,
     uploadPreviewActivityIndicator: colors.gray1,
+    dropUIBG: 'rgba(255, 255, 255, 0.96)',
```

Comment:
We are working on updating the app theme and it is suggested not to add new variables until that is done. But I am fine with this one for now.

Reply:
@shawnborton what should be the new value of this theme color?

Reply:
I think we should just slightly transparentize the current dark app BG color.

Reply:
I vaguely recall we discussed using a color function here rather than needing a hard rgba value. Is that still a thing?